constraints.txt

```diff
@@ -0,0 +1,2 @@
+git+git://github.com/david4096/ga4gh-client.git@protobuf31#egg=ga4gh_client
```

Comment:
"Our internal dependencies shouldn't have stuck versions in the requirements.txt unless it is a released version."

So the release manager is going to have to remember this, and track the versions down that s/he is using, on every release?

"We could do rolling releases to pypi to bump the version number with each accepted PR"

This seems like somewhere between ambitious and problematic.  Pypi releases are for everyone (i.e. end users), not just for developers.  In any event, it's a different conversation than what this thread is addressing.